README.md

```diff
@@ -9,11 +9,10 @@ My first project during the first week of training in Microverse.
 - CSS
 - Linters
 ​
-​
-​
 ## Getting Started
 ​
-To get a local copy up and running follow these simple example steps.
+git clone https://github.com/Epaltechs/Hello-World.git
+cd Hello-World
 ​
 ## Authors
 
```

Comment:
- [ ] OPTIONAL: I nocticed that the Authors section is empty, please consider adding your social links in Authors section eg: 

```
## Author
👤 **Your_name**

- GitHub: [@gihub_name](githib_url)
- LinkedIn: [linkedin_name](linkedin_url)

![Epaltechs's GitHub stats](https://github-readme-stats.vercel.app/api?username=epaltechs&count_private=true&theme=dark&show_icons=true)

```